examples/angular/src/lib/shorten/BUILD.bazel

```diff
@@ -6,6 +6,7 @@ package(default_visibility = ["//:__subpackages__"])
 # with a main field to show how to create a first-party npm lib with a package.json
 ts_library(
     name = "shorten",
+    package_name = "@bazel/shorten",
     srcs = ["index.ts"],
     module_name = "@bazel/shorten",
```

Comment:
should the `module_name` be removed now? or is that still required as the AMD module name?

Reply:
can't yet at it is tied to ts_library internals; this PR makes it so that module_name no longer turns on the linker as it was causing issues downstream